exchange/order/limits/limits_types.go

```diff
@@ -0,0 +1,74 @@
+package limits
+
+import (
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common/key"
+)
+
+// Public errors for order limits
+var (
+	ErrCannotLoadLimit         = errors.New("cannot load limit, levels not supplied")
+	ErrOrderLimitNotFound      = errors.New("order limit not found")
+	ErrExchangeLimitNotLoaded  = errors.New("exchange limits not loaded")
+	ErrPriceBelowMin           = errors.New("price below minimum limit")
+	ErrPriceExceedsMax         = errors.New("price exceeds maximum limit")
+	ErrPriceExceedsStep        = errors.New("price exceeds step limit") // price is not divisible by its step
```

Comment:
🐒   Seems comment is more parsable than the error, so maybe better
And probably rename error too. I'd favour `Mod` since we all know it.
Alternative `HasStepRem`
```suggestion
	ErrPriceHasStepMod        = errors.New("price not multiple of step limit")
```